website/docs/user-guides/image-registry.md

```diff
@@ -3,5 +3,54 @@ id: image-registry
 title: Using different Image Registries in a Workflow
 sidebar_label: Using different Image Registries
 ---
+---
+
+## Image Registry
+A container image registry can be defined as a collection of repositories that store container image.
+These can be either public or private. Few of the container image registries are Docker, Red Hat Quay, Google Container Registry. 
+By default LitmusChaos uses DockerHub for managing the different images. These images are then used in Chaos Workflows. Few images that are used in the Litmus workflows are `litmuschaos:k8s`, `litmuschaos:litmus-checker` etc.
+With ChaosCenter, you get the privilege to use your own/custom image registries for Chaos Workflows.
+
```

Comment:
Can you please add a `Before you begin` section where the associated concepts that are required as a pre-requisite will be referenced!